### PR TITLE
ns for fx: clean up convtranspose mappings

### PR DESCRIPTION
**Pull request by vkuzo**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #76993 ns for fx: remove duplicated BNReLU mappings
* #76992 ns for fx: remove quantized ReLU6 from mapping
* #76991 ns for fx: remove GroupNorm from mapping
* #76990 ns for fx: clean up linear in relationship mapping
* #76989 ns for fx: make torch.ops.quantized.dropout mapping dynamic
* #76988 ns for fx: remove torch.ops.quantized.cat
* **#76980 ns for fx: clean up convtranspose mappings**
* #76978 ns for fx to backend_config_dict [2/x]: native lowering mappings
* #76958 ns for fx to backend_config_dict [1/x]: fused and qat modules

Summary:

Fixes a couple of problems with `ConvTranspose` in NS mappings:
1. deletes the dynamic versions, as they do not work yet
2. deletes `ConvTranspose3d`, as it's not swapped yet in the quantization workflow
3. removes a duplicate set

Test plan:

```
python test/test_quantization.py -k FXGraphMatcher
python test/test_quantization.py -k FXNumericSuite
```

Differential Revision: [D36212234](https://our.internmc.facebook.com/intern/diff/D36212234)

### CODE REVIEW

**Comment by vkuzo** [comment]
@vkuzo has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D36212234).

File: torch/ao/ns/fx/mappings.py [hunk 1]
```diff
@@ -28,15 +28,12 @@ def get_base_name_to_sets_of_related_ops() -> Dict[str, Set[NSNodeTargetType]]:
         # conv modules
         set([
             nn.Conv1d,
-            nnqd.Conv1d,
```
Comment (jerryzh168) [inline torch/ao/ns/fx/mappings.py:31]: where are these covered? didn't find them in the module mappings in https://github.com/pytorch/pytorch/blob/master/torch/ao/quantization/fx/_lower_to_native_backend.py

Comment (jerryzh168) [reply torch/ao/ns/fx/mappings.py:31]: oh, just saw it's deleted because it does not work yet

**Review by jerryzh168 — APPROVED** [review]

**Comment by vkuzo** [comment]
@vkuzo has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D36212234).

**Comment by vkuzo** [comment]
@vkuzo has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D36212234).

**Comment by vkuzo** [comment]
@vkuzo has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D36212234).

**Comment by vkuzo** [comment]
@vkuzo has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D36212234).

**Comment by vkuzo** [comment]
@vkuzo has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D36212234).

**Comment by vkuzo** [comment]
@vkuzo has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D36212234).
